recipes/dscribe/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "dscribe" %}
+{% set version = "0.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7f232e6572a9815d232003e85ba714235cd203b9b3a7749b7133c28acf325b3e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
```

The `setup.py` claims it should work with any Python 3 version:
```suggestion
  skip: True  # [py2k]
```
(It's probably wrong as it even claims 3.0 but it's usual for 3.7 to work and not 3.6.)

---

On Python 3.6 it has issues to resolve the dependencies, therefore I restricted the package to 3.7.

---

That's good enough for me, LGTM 👍